package.json

```diff
@@ -61,6 +61,7 @@
     "eslint-plugin-promise": "^5.1.0",
     "express": "^4.16.4",
     "express-bearer-token": "^2.4.0",
+    "express-cache-on-demand": "^1.0.2",
```

Comment:
On release day let's make sure we bump this to the new, debugged release of `express-cache-on-demand` as a minimum just to be thorough.